org.martchus.syncthingtray.yml

```diff
@@ -0,0 +1,147 @@
+id: org.martchus.syncthingtray
```

Comment:
`martchus.org` is NXDOMAIN (doesn't exist)

Yet:

> The domain used for the application ID is controlled by the application developers either directly or through the code hosting (e.g. GitHub, GitLab, SourceForge, etc.). The [application id guidelines](https://docs.flathub.org/docs/for-app-authors/requirements#application-id) are followed.

Reply:
So you're basically saying the app ID minus the last part needs to be an existing domain controlled by the author (in this case me). Then I assume it needed to be `com.github.syncthingtray` as code is hosted on https://github.com/Martchus/syncthingtray - although this would lack the "martchus" part (as my name is only part of the URL path). Considering https://martchus.github.io/syncthingtray exists I suppose `io.github.martchus.syncthingtray` would also fulfill your requirement and might be the better option.

Reply:
it's all in the documentation linked from that item you acknowledged.

The checkbox are NOT a captcha.

Reply:
Well, I haven't checked anything myself, just saw this coming up in my GitHub notifications :-)

Reply:
It's my bad just not check it carefully enough 😘